docs/website/docs/general-usage/data-enrichments/user_agent_device_data_enrichment.md

```diff
@@ -230,19 +213,11 @@ The first step is to register on [SerpAPI](https://serpapi.com/) and obtain the
    - Transformer function
 
 
-   The `dlt` library's `transformer` and `add_map` functions serve distinct purposes in data
-   processing.
+   The `dlt` library's `transformer` and `add_map` functions serve distinct purposes in data processing.
 
-   `Transformers` used to process a resource and are ideal for post-load data transformations in a
-   pipeline, compatible with tools like `dbt`, the `dlt SQL client`, or Pandas for intricate data
-   manipulation. To read more:
-   [Click here.](../../general-usage/resource#process-resources-with-dlttransformer)
+   `Transformers` are used to process a resource and are ideal for post-load data transformations in a pipeline, compatible with tools like `dbt`, the `dlt SQL client`, or Pandas for intricate data manipulation. To read more: [Click here.](../../general-usage/resource#process-resources-with-dlttransformer)
 
-   Conversely, `add_map` used to customize a resource applies transformations at an item level
-   within a resource. It's useful for tasks like anonymizing individual data records. More on this
-   can be found under
-   [Customize resources](../../general-usage/resource#customize-resources) in the
-   documentation.
+   Conversely, `add_map` is used to customize a resource and applies transformations at an item level within a resource. It's useful for tasks like anonymizing individual data records. More on this can be found under [Customize resources](../../general-usage/resource#customize-resources) in the documentation.
```

Comment:
```suggestion
   Conversely, `add_map` is used to customize a resource and applies transformations at an item level within a resource. It's useful for tasks like anonymizing individual data records. More on this can be found under [customize resources](../../general-usage/resource#customize-resources) in the documentation.
```